.github/update_docs.py

```diff
@@ -14,11 +15,25 @@ def main(root: Path = typer.Argument(Path.cwd(), help="Root path to look in")):
     auto-replaced).
     """
     msg.info(f"Updating auto-generated docs in {root}")
+    entries = []
     # We look specifically for project directories
     for path in root.glob(f"**/*/{PROJECT_FILE}"):
         path = path.parent
         project_document(path, path / "README.md")
 
+        # prep data for the json file
+        config = load_project_config(path)
+        entry = {"shortname": f"{path.parent.name}/{path.name}"}
+        entry["title"] = config["title"]
+        entry["description"] = config.get("description", "")
+        entries.append(entry)
+
+
+    with open("projects.jsonl", "w") as jsonfile:
+        for entry in entries:
+            jsonfile.write(json.dumps(entry))
+            jsonfile.write("\n")
```

Comment:
It would be nice to print a message to the command line that acknowledges this has happened, similar to how the rest of the command works